Quadriga/src/main/java/edu/asu/spring/quadriga/dao/workspace/impl/WorkspaceDAO.java

```diff
@@ -161,8 +161,11 @@ public boolean deleteWorkspace(String wsId) {
             throws QuadrigaStorageException {
         List<WorkspaceDTO> workspaceDTOList = null;
         try {
-            Query query = sessionFactory.getCurrentSession().createQuery(
-                    "Select distinct projWork.workspaceDTO from ProjectWorkspaceDTO projWork INNER JOIN projWork.workspaceDTO.workspaceCollaboratorDTOList workcollab where workcollab.quadrigaUserDTO.username =:username and projWork.projectDTO.projectid =:projectid and projWork.workspaceDTO.isarchived =:isarchived and projWork.workspaceDTO.isdeactivated =:isdeactivated");
+            Query query = sessionFactory.getCurrentSession()
+                    .createQuery("Select distinct projWork.workspaceDTO from ProjectWorkspaceDTO projWork INNER JOIN "
+                            + "projWork.projectDTO.projectCollaboratorDTOList projcollab where projWork.workspaceDTO.createdby !=:username and projWork.projectDTO.projectid =:projectid "
+                            + "and projWork.workspaceDTO.isarchived =:isarchived and projWork.workspaceDTO.isdeactivated =:isdeactivated");
+
```

Comment:
This is the query I changed to properly get "activeCollaboratingWorkSpaces" of user.